src/components/Genome3D/List/index.js

```diff
@@ -157,6 +157,10 @@ export const List = (
               </Link>
             )}
             isSearchable={true}
+            customiseSearch={{
+              validation: /^([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2})$/i,
+              message: 'Accepts only UniProt accessions',
```

Comment:
It took me a while to find where this message was appearing, maybe using a tooltip that displays an icon at the side of the textbox is more obvious